examples/platform/linux/Options.cpp

```diff
@@ -173,7 +174,7 @@ OptionDef sDeviceOptionDefs[] = {
 #endif // CHIP_DEVICE_CONFIG_ENABLE_WIFIPAF
 #endif // CHIP_DEVICE_CONFIG_ENABLE_WIFI
 #if CHIP_ENABLE_OPENTHREAD
-    { "thread", kNoArgument, kDeviceOption_Thread },
+    { "thread", kArgumentOptional, kDeviceOption_Thread },
```

Comment:
the naming here for `thread` and `mThread` is not intuitive to me ... what is a `thread` ? Maybe `thread-node-id` and `mThreadNodeId` if this is its usage?